blog/fine-grained-news-2025-09.md

```diff
@@ -0,0 +1,114 @@
+title: Fine Grained News - September 2025
+description: Fine Grained News
+slug: fine-grained-news-2025-09
+date: 2025-09-05
+authors: aaguiar
+tags: [newsletter]
+image: https://openfga.dev/img/og-rich-embed.png
+hide_table_of_contents: false
+---
+# Fine Grained News - September 2025
+
+After a long hiatus, we are back with Fine Grained News! The best way to keep up to date with the OpenFGA community! 
+
+First of all, we want to thank the OpenFGA community for helping OpenFGA get beyond 4k stars! 
+
+## What have we been up to
+
+### OpenFGA Performance
+
+The OpenFGA team has been mostly focused on performance improvements, and we are close to finishing a big round of improvements. 
+
+- The `/check` endpoint is now **up to 10x faster** in some scenarios
+- The `/list-objects` endpoint is **up to 5x faster** in some scenarios (the enhancements are still behind an experimental flag).
+- The caching implementation has a more sophisticated eviction mechanism that allows OpenFGA to cache entries for a longer time
+
+### Improved Write API
+
+OpenFGA v1.10 adds additional parameters to the [`/write`](https://openfga.dev/api/service#/Relationship%20Tuples/Write) endpoint that allow specifying the behavior when duplicated tuples are written or non-existing tuples are deleted. 
+
+The SDKs and the CLI do not support the new options yet.
+
+### Terraform Provider
+
+[Maurice Ackel](https://www.linkedin.com/in/maurice-ackel/) contributed the official [OpenFGA Terraform Provider](https://github.com/openfga/terraform-provider-openfga). Take a look at it, and let Maurice know if you are using it!
+
+### SDKs and Tooling
+
+- We shipped several improvements across all SDKs:
+
+  - Improved how the SDKs retry when getting 429 errors
+  - BatchCheck support (.NET coming soon!)
+  - .NET Standard 2.0 support for .NET is in Alpha, and will be released soon.
+
+- The CLI has several improvements:
+
+  - Supports running tests using glob patterns (`fga model test --tests *.yaml`)
+  - Supports `jsonl` format for tuples
+  - Allows specifying multiple tuple files in store tests
+  - Enables grouping users/objects in store tests, avoiding duplication:
+
+```
+    - object: group:employees
+      users:
+        - user:1
+        - user:2
+      assertions:
+        member: true
+
+    - objects:
+        - group:admins
+        - group:employees
+      user: user:1
+      assertions:
+          member: true
+
+```
+<!-- markdown-link-check-disable -->
+
+  - We've been playing with an OpenFGA MCP to help with modeling. You can add the MCP by pointing your MCP client (e.g., VS Code) to https://mcp.openfga.dev/mcp. You can find the code [here](https://github.com/aaguiarz/openfga-modeling-mcp), or see it in action [here](https://www.youtube.com/watch?v=JNBtf-1NrPM). It's pretty awesome! :)
+<!-- markdown-link-check-enable -->
+
+## New OpenFGA Adopters
+
+The list of companies/projects keeps increasing! These were added since the last edition of Fine Grained News:
```

Comment:
```suggestion
The list of companies and projects that have [adopted OpenFGA](https://github.com/openfga/community/blob/main/ADOPTERS.md) keeps increasing - with the following added since the last Fine-Grained News was published:
```

Just to highlight adopters. "The list of companies/projects keeps increasing!" doesn't provide a lot of context for those who missed the header.